src/Global/molecules/UserMenu.tsx

```diff
@@ -186,6 +251,14 @@ const UserMenu: React.FC<IUserMenu> = ({
               Language / 言語
             </LanguageMenu>
         }
+        {(userDrawerFooter?.icon !== '' || userDrawerFooter?.title !== '' ) ?
+          <FooterContainer title={userDrawerFooter?.title}>
+            <Icon icon='Information' size={14} color='dimmed' />
```

Comment:
You are checked for an `icon` value for this block but the icon here is hard coded. Is this intended to use the prop? You can use `Information` for a fallback value if needed.

Reply:
Done